src/main/java/org/folio/common/ListUtils.java

```diff
@@ -21,7 +21,7 @@ public static String createPlaceholders(int size) {
     return String.join(",", Collections.nCopies(size, "?"));
   }
 
-  public static  String createInsertPlaceholders(int size) {
-    return String.join(",", Collections.nCopies(size,"(?,?)"));
+  public static  String createInsertThreePlaceholders(int size) {
```

Comment:
What if we need 4 placeholders or 5?

Reply:
@dmtkachenko refactored method to be more flexible